static/js/search.js

```diff
@@ -0,0 +1,73 @@
+    window.renderGoogleSearchResults = () => {
```

Comment:
I know Google is blocked, so we may also want to change this function name to remove doubt for contributors in the future